src/pages/registration/containers/RegistrationContainer.tsx

```diff
@@ -34,17 +32,7 @@ export function RegistrationContainer(): JSX.Element | null {
                 return <CourseContainer course={course} />;
               })
           ) : (
-            <>
-              <Divider my="4" />
-              <Container alignItems="center" maxW="container.xl">
-                <Heading size="md" color={mode('gray', 'dark.header')}>
-                  Unable to find saved courses for{' '}
-                  <Text as="span" color={mode('black', 'white')}>
-                    {getReadableTerm(term)}
-                  </Text>
-                </Heading>
-              </Container>
-            </>
+            <RegistrationNotFound />
```

Comment:
After you rename and move it to `common` you can add it to the booklist page too. Right now it says `unable to find courses or textbooks...` on that page, but I think we should separate those so the user actually knows. I wouldn't add that logic to your component though, just add your component to that page for the case no courses are found, then keep the old message (minus the course part) for if no textbooks are found.